mlrun/model_monitoring/db/tsdb/base.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+from abc import ABC
+
+import pandas as pd
+
+
+class TSDBConnector(ABC):
+    def __init__(self, project: str):
+        """
```

Comment:
The constructor isn't normally documented, and the relevant information goes into the class docstring instead.